packages/client/test/unit/Validator.test.ts

```diff
@@ -0,0 +1,86 @@
+import 'reflect-metadata'
+import { Wallet } from '@ethersproject/wallet'
+import { EthereumAddress, toStreamID, toStreamPartID } from 'streamr-client-protocol'
+import { Stream } from '../../src/Stream'
+import { StreamRegistry } from '../../src/registry/StreamRegistry'
+import { StreamRegistryCached } from '../../src/registry/StreamRegistryCached'
+import { Validator } from '../../src/Validator'
+import { createMockMessage, mockContext } from '../test-utils/utils'
+import { STREAM_CLIENT_DEFAULTS, SubscribeConfig } from '../../src/Config'
+
+const publisherWallet = Wallet.createRandom()
```

Comment:
```suggestion
const publisherWallet = fastWallet()
```
for faster test runtime